src/main/java/dev/peev/pastebox/domain/entities/Paste.java

```diff
@@ -20,6 +20,9 @@
   @Column(name = "content", columnDefinition = "text")
   private String content;
 
+  @Column(name = "syntax", columnDefinition = "text")
```

Comment:
`columnDefinition = "text"` is obsolete here as all the syntaxis will be <255 chars long. you can remove it

Reply:
done